src/test/kotlin/com/terraformation/backend/tracking/db/ObservationStoreTest.kt

```diff
@@ -1871,4 +1874,169 @@ class ObservationStoreTest : DatabaseTest(), RunsAsUser {
           "Coordinates after update")
     }
   }
+
+  @Nested
+  inner class RemovePlotFromTotals {
+    @Test
+    fun `updates observed species totals across all observations`() {
+      val speciesId1 = insertSpecies()
+      val speciesId2 = insertSpecies()
+      val speciesId3 = insertSpecies()
+
+      val zoneId1 = insertPlantingZone()
+      insertPlantingSubzone()
+      val zone1PlotId1 = insertMonitoringPlot()
+      val zone1PlotId2 = insertMonitoringPlot()
+      val zoneId2 = insertPlantingZone()
+      insertPlantingSubzone()
+      val zone2PlotId1 = insertMonitoringPlot()
+
+      val observationId1 = insertObservation(completedTime = Instant.ofEpochSecond(1))
+
+      helper.insertObservationScenario(
+          ObservationZone(
+              zoneId = zoneId1,
+              plots =
+                  listOf(
+                      ObservationPlot(
+                          zone1PlotId1,
+                          listOf(
+                              PlantTotals(speciesId1, live = 2, dead = 1, existing = 1),
+                              PlantTotals(speciesId2, live = 1),
+                              PlantTotals("fern", live = 1, dead = 1),
+                          ),
+                      ),
+                      ObservationPlot(
+                          zone1PlotId2,
+                          listOf(
+                              PlantTotals(speciesId2, live = 1, dead = 1),
+                          )))),
+          ObservationZone(
+              zoneId = zoneId2,
+              plots =
+                  listOf(
+                      ObservationPlot(
+                          zone2PlotId1,
+                          listOf(
+                              PlantTotals(speciesId1, live = 1),
+                              PlantTotals(speciesId3, live = 1),
+                          )))))
+
+      val observationId2 = insertObservation(completedTime = Instant.ofEpochSecond(2))
+
+      helper.insertObservationScenario(
+          ObservationZone(
+              zoneId = zoneId1,
+              plots =
+                  listOf(
+                      ObservationPlot(
+                          zone1PlotId1,
+                          listOf(
+                              PlantTotals(speciesId1, live = 1, dead = 1, existing = 1),
+                              PlantTotals(speciesId2, live = 1),
+                              PlantTotals("fern", live = 2),
+                          ),
+                      ),
+                      ObservationPlot(
+                          zone1PlotId2,
+                          listOf(
+                              PlantTotals(speciesId2, live = 1),
+                          ))),
+          ))
+
+      store.removePlotFromTotals(zone1PlotId1)
+
+      helper.assertTotals(
+          setOf(
+              ObservedPlotSpeciesTotalsRow(
```

Comment:
Okay so this is verifying that despite removing stats from overall totals, the observation data is maintained, right?